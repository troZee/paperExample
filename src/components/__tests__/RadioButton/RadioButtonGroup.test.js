import React from 'react';
import renderer from 'react-test-renderer';
import RadioButton from '../../RadioButton/RadioButton';

describe('RadioButtonGroup', () => {
  it('renders properly', () => {
    const tree = renderer
      .create(
        <RadioButton.Group value="first" onValueChange={() => {}}>
          <RadioButton value="first" />
        </RadioButton.Group>
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
