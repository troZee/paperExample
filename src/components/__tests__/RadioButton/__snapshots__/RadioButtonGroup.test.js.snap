// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RadioButtonGroup renders properly 1`] = `
<View
  accessibilityRole="button"
  accessibilityStates={Array []}
  accessible={true}
  focusable={true}
  isTVSelectable={true}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Array [
      Object {
        "overflow": "hidden",
      },
      Object {
        "borderRadius": 18,
        "padding": 6,
      },
    ]
  }
>
  <View
    style={
      Object {
        "opacity": 1,
      }
    }
  >
    <Text
      accessibilityElementsHidden={true}
      allowFontScaling={false}
      importantForAccessibility="no-hide-descendants"
      pointerEvents="none"
      style={
        Array [
          Object {
            "color": "#03dac4",
            "fontSize": 24,
          },
          Array [
            Object {
              "transform": Array [
                Object {
                  "scaleX": 1,
                },
              ],
            },
            Object {
              "backgroundColor": "transparent",
            },
          ],
          Object {
            "fontFamily": "Material Design Icons",
            "fontStyle": "normal",
            "fontWeight": "normal",
          },
          Object {},
        ]
      }
    >
      ï„¬
    </Text>
  </View>
</View>
`;
